### PR TITLE
[5.11.0] Update sections with callback regex configuration

### PR DESCRIPTION
**Pull request by mpmadhavig**
## Purpose
$subject 

The changes are added as notes in each section. Sections changed are as below.

1. Self Registration
2. Lite User Registration
3. Username recovery
4. Password recovery

### Related Issues
- https://github.com/wso2/docs-is/issues/3691

### CODE REVIEW

File: en/docs/administer/product-level-security-guidelines.md [hunk 1]
```diff
@@ -197,13 +197,12 @@ By default, XSS attacks are prevented in the latest WSO2 Identity Server version
...
 
-For password recovery, you can define a regular expression to validate the callback URL. The default configuration 
-allows any callback URL. Note that if you are using the recovery option, it is highly recommended to define the regular 
-expression that validates and only allows access to specific callback URLs.
+For the below mentioned flow, you can define a regular expression to validate the callback URL. The default configuration allows any callback URL. Note that if you are using the recovery option, it is highly recommended to define the regular expression that validates and only allows access to specific callback URLs.
 
-!!! info
-
-    For more informatoin, see [Password Recovery](../../learn/password-recovery).
+1. [Password Recovery](../../learn/password-recovery)
```
Comment (nilminiwso2) [inline en/docs/administer/product-level-security-guidelines.md:202]: These should be bullet point not numbers as we are providing a list and not a procedure.

File: en/docs/learn/configuring-user-name-recovery.md [hunk 2]
```diff
@@ -80,7 +80,10 @@ Follow the steps below to configure WSO2 Identity Server to enable username reco
 	-	**Username recovery**
 	-	**Manage notifications sending internally**
 
-	<img src="../../assets/img/learn/user-name-recovery-options.png" alt="User Name Recover options" width="600" style="border:1px solid grey">  	
+	<img src="../../assets/img/learn/user-name-recovery-options.png" alt="User Name Recover options" width="600" style="border:1px solid grey">		
+
+	!!! note
+        - The recommended  **Recovery callback URL regex** to use when testing the product is `^https:\/\/localhost:9443\/.*`. But user should modify it meet their requirements when they are deploying the product.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-user-name-recovery.md:86]:
```suggestion
        - The recommended  **Recovery callback URL regex** to use when testing the product is `^https:\/\/localhost:9443\/.*`. But users should modify it to meet their requirements when they deploy the product.
```

File: en/docs/learn/password-recovery.md [hunk 3]
```diff
@@ -47,6 +47,9 @@ with email notification.
 .
 
     ![notification-based-password-recovery](../assets/img/using-wso2-identity-server/notification-based-password-recovery-conf.png)
+
+    !!! note
+        - The recommended  **Recovery callback URL regex** to use when testing the product is `^https:\/\/localhost:9443\/.*`. But user should modify it meet their requirements when they are deploying the product.
```
Comment (nilminiwso2) [inline en/docs/learn/password-recovery.md:52]:
```suggestion
        - The recommended  **Recovery callback URL regex** to use when testing the product is `^https:\/\/localhost:9443\/.*`. However, users should modify it to meet their requirements when they deploy the product.
```

File: en/docs/learn/self-registration-and-account-confirmation.md [hunk 4]
```diff
@@ -121,6 +121,15 @@ which is `         carbon.super        `.
     </div></div> 
     </div></td>
     </tr>
+    <tr class="even">
+    <td>User self registration callback URL regex</td>
+    <td>The regex pattern used to validate the callback URL you are adding as the recovery flow callback url.
```
Comment (nilminiwso2) [inline en/docs/learn/self-registration-and-account-confirmation.md:126]:
```suggestion
    <td>The regex pattern used to validate the callback URL you are adding as the recovery flow callback URL. 
```

File: en/docs/learn/self-registration-and-account-confirmation.md [hunk 5]
```diff
@@ -121,6 +121,15 @@ which is `         carbon.super        `.
     </div></div> 
     </div></td>
     </tr>
+    <tr class="even">
+    <td>User self registration callback URL regex</td>
+    <td>The regex pattern used to validate the callback URL you are adding as the recovery flow callback url. 
+    
+    <div class="admonition note">
+    <p class="admonition-title">Note</p>
+    <p>The sample <b>User self registration callback URL regex</b> to use when testing the product is <code>^https:\/\/localhost:9443\/.*</code>. But user should modify it meet their requirements when they are deploying the product.</td></p>
```
Comment (nilminiwso2) [inline en/docs/learn/self-registration-and-account-confirmation.md:130]:
```suggestion
    <p>The sample <b>self registration callback URL regex</b> to use when testing the product is <code>^https:\/\/localhost:9443\/.*</code>. However, users should modify it to meet their requirements when they deploy the product.</td></p>
```

File: en/docs/learn/username-recovery.md [hunk 6]
```diff
@@ -79,6 +79,10 @@ with the WSO2 IS.
 
 5.  Select the **Enable Username Recovery** checkbox and the **Enable
     Notification Internally Management** checkbox.  
+    
+    !!! note
+        - The recommended  **Recovery callback URL regex** to use when testing the product is `^https:\/\/localhost:9443\/.*`. But user should modify it meet their requirements when they are deploying the product.
```
Comment (nilminiwso2) [inline en/docs/learn/username-recovery.md:84]:
```suggestion
        The recommended  **Recovery callback URL regex** to use when testing the product is `^https:\/\/localhost:9443\/.*`. However, users should modify it to meet their requirements when they deploy the product.
```

File: en/docs/administer/product-level-security-guidelines.md [hunk 7]
```diff
@@ -197,13 +197,12 @@ By default, XSS attacks are prevented in the latest WSO2 Identity Server version
 
 ## Callback URL Regular Expressions
 
-For password recovery, you can define a regular expression to validate the callback URL. The default configuration 
-allows any callback URL. Note that if you are using the recovery option, it is highly recommended to define the regular 
-expression that validates and only allows access to specific callback URLs.
+For the below mentioned flow, you can define a regular expression to validate the callback URL. The default configuration allows any callback URL. Note that if you are using the recovery option, it is highly recommended to define the regular expression that validates and only allows access to specific callback URLs.
```
Comment (nilminiwso2) [inline en/docs/administer/product-level-security-guidelines.md:200]:
```suggestion
For the below-mentioned flow, you can define a regular expression to validate the callback URL. The default configuration allows any callback URL. Note that if you are using the recovery option, it is highly recommended to define the regular expression that validates and only allows access to specific callback URLs.
```

File: en/docs/administer/product-level-security-guidelines.md (hunk 1 above)
Comment (mpmadhavig) [reply en/docs/administer/product-level-security-guidelines.md]: Resolved with 620dac406590a635da589dc3ba3a5affbd2c8053

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**